tests/test_publisher_client.py

```diff
@@ -0,0 +1,192 @@
+import os
+import unittest
+
+from uid2_client import Uid2PublisherClient
+from uid2_client import TokenGenerateInput
+from uid2_client import TokenGenerateResponse
+from uid2_client.identity_tokens import IdentityTokens
+from urllib.request import HTTPError
+
+
+class PublisherEuidIntegrationTests(unittest.TestCase):
+
+    EUID_SECRET_KEY = None
+    EUID_API_KEY = None
+    EUID_BASE_URL = None
+
+    publisher_client = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.EUID_BASE_URL = os.getenv("EUID_BASE_URL")
+        cls.EUID_API_KEY = os.getenv("EUID_API_KEY")
+        cls.EUID_SECRET_KEY = os.getenv("EUID_SECRET_KEY")
+
+        print(cls.EUID_BASE_URL, cls.EUID_API_KEY, cls.EUID_SECRET_KEY)
+
+        if cls.EUID_BASE_URL and cls.EUID_API_KEY and cls.EUID_SECRET_KEY:
```

Comment:
adeed both but used "raise Exception" instead of "print" to halt testing before another exception is thrown. That way it is more clear what the issue is.

Reply:
i didn't see a new commit that does what you said above?

Reply:
commited and push the change now